api/Process/ContaoConsole.php

```diff
@@ -169,6 +169,68 @@ public function checkDatabaseMigrations(): ?array
         ];
     }
 
+    public function getUsers(): ?array
+    {
+        $commands = $this->getCommandList();
+
+        if (
+            !isset($commands['contao:user:list']['options'])
+            || !\in_array('format', $commands['contao:user:list']['options'], true)
+            || !\in_array('column', $commands['contao:user:list']['options'], true)
+        ) {
+            return null;
+        }
+
+        $arguments = [
+            'contao:user:list',
+            '--format=json',
+            '--column=username',
+            '--column=name',
+            '--column=admin',
+            '--column=dateAdded',
+            '--column=lastLogin',
+            '--no-interaction',
```

Comment:
I didn’t load all columns and didn’t use `--admins` because of https://github.com/contao/contao/issues/4937